ServerRoomMonitoring.Api/Models/SensorMessage.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace ServerRoomMonitoring.Api.Models
+{
+    public class SensorMessage
```

Comment:
Jeśli to jest taki sam obiekt jak w Generatorze, to może trafić do osobnego projektu tylko z encjami, który byłby zależnością API i Generatora. Dzieki czemu nie trzeba by zmieniać tej klasy w obu projektach.